pkg/telemetrylogs/condition_builder.go

```diff
@@ -99,7 +99,12 @@ func (c *conditionBuilder) conditionFor(
 		return sb.NotILike(tblFieldName, fmt.Sprintf("%%%s%%", value)), nil
 
 	case qbtypes.FilterOperatorRegexp:
-		return fmt.Sprintf(`match(%s, %s)`, tblFieldName, sb.Var(value)), nil
+		// check if the field is fullText, if yes then convert value to lower
+		valueStr := sb.Var(value)
+		if tblFieldName == DefaultFullTextColumn.Name {
+			valueStr = fmt.Sprintf("LOWER(%s)", valueStr)
+		}
```

Comment:
Can we move this logic to https://github.com/SigNoz/signoz/blob/eca13075e9f40f0ee2e001056f3cc6f536fc902e/pkg/telemetrylogs/condition_builder.go#L61-L68?

I think, then you can also remove the need to define `LOWER(body)` as column/field and the fields list can be just a list of actual columns on table.